pkg/apis/packet/v1alpha1/types_controlplane.go

```diff
@@ -24,4 +24,11 @@ import (
 // ControlPlaneConfig contains configuration settings for the control plane.
 type ControlPlaneConfig struct {
 	metav1.TypeMeta `json:",inline"`
+
+	Persistence *Persistence `json:"persistence,omitempty"`
+}
+
+// PersistenceConfig contains configuration options for shoot persistent storage.
+type Persistence struct {
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
doc string missing

Also, it seems to be an optional field, so the `// +optional` tag should be added and `bool` should become `*bool`.

Reply:
Done